src/test/java/uk/gov/pay/connector/gateway/worldpay/wallets/WorldpayWalletAuthorisationHandlerTest.java

```diff
@@ -110,16 +111,41 @@ public void shouldSendGooglePay3DSRequestWhenGooglePayDetailsWithoutIpAddressAre
         }
     }
 
+    @Test
+    public void shouldSendGooglePay3DSRequestWhenGooglePayDetailsWithIpAddressArePresent() throws Exception {
```

Comment:
Can we use `3ds` instead of `3DS` for consistency?

```suggestion
    public void shouldSendGooglePay3dsRequestWhenGooglePayDetailsWithIpAddressArePresent() throws Exception {
```

Reply:
Done.